psi4/driver/driver_nbody.py

```diff
@@ -304,13 +304,17 @@ def nbody_gufunc(func, method_string, **kwargs):
             energies_dict[pair] = core.get_variable("CURRENT ENERGY")
             core.print_out("\n       N-Body: Complex Energy (fragments = %s, basis = %s: %20.14f)\n" %
                                                                 (str(pair[0]), str(pair[1]), energies_dict[pair]))
-
             # Flip this off for now, needs more testing
             #if 'cp' in bsse_type_list and (len(bsse_type_list) == 1):
             #    core.set_global_option('DF_INTS_IO', 'LOAD')
 
             core.clean()
 
+    # Set vairables for N-Body intermediates
+    for pair in energies_dict.keys():
+        core.set_variable("N-BODY (%s)@(%s) TOTAL ENERGY" %
```

Comment:
Did you find a handy `wfn` to attach these to, too?

Reply:
But you're not adding your psivars to `energy_body_dict` to be subsequently attached to `wfn`, right? If say, the `psi4.get_variable('your psivar')` in the tests got its psivars from `wfn`, not `P::e`, it wouldn't find it currently, I think. Future-proof by making that test pass no matter which loc psivars are coming from.

Reply:
That's correct, yes.  I will add them to `energies_dict` so that when they get attached to the wavefunction with `wfn.nbody_energy = energies_dict` they are recognized.